terraform/projects/prom-ec2/paas-production/main.tf

```diff
@@ -64,6 +64,10 @@ data "pass_password" "prometheus_htpasswd" {
   path = "prometheus-basic-auth-htpasswd"
 }
 
+data "pass_password" "dm_elasticsearch_metrics_password" {
+  path = "dm-elasticsearch-metrics-password"
```

Comment:
We'll need to set this path correctly once the new secret is in your `pass` store.

Reply:
Done in https://github.com/alphagov/re-secrets/commit/75c91ca4e0051f70d9625f3da71a9af2484d5a5c and this won't need changing.